src/devices/cpu/dsp56156/inst.h

```diff
@@ -533,15 +514,10 @@ class Bcc: public Instruction
 		decode_cccc_table(BITSn(word0,0x000f), m_mnem);
 		return true;
 	}
-	void disassemble(std::string& retString) const override
+	std::string disassemble() const override
 	{
-		std::string opcode = "b" + opMnemonicAsString(m_mnem);
-		// NEW // sprintf(opcode_str, "b.%s", M);
-
-		char temp[32];
-		sprintf(temp, ">*+$%x", 2 + m_immediate);
-		// NEW // sprintf(temp, "$%04x (%d)", pc + 2 + (int16_t)word1, (int16_t)word1);
-		retString = opcode + " " + std::string(temp);
+		auto opcode = "b" + opMnemonicAsString(m_mnem);
+		return opcode + util::string_format(" >*+$%x", 2 + m_immediate);
```

Comment:
Why not use a single `util::string_format` and avoid the extra temporaries and concatenations?